tests/Makefile

```diff
@@ -53,16 +53,11 @@ NC='\033[0m'
 
 .PHONY: clean list build all
 
-all: framework build 
+all: framework
 
 build:
 	@echo -e ${GREEN}Build [$@] ${NC}
 	@make -C ../ test TARGET=$(TARGET)
-	@echo -e ${GREEN}Copy Assets to [${BIN_DIR}/assets] ${NC}
```

Comment:
this is required so you can run the binary why would you remove this?

Reply:
Currently, with recursive make in framework of ut-core Makefile, make test gets called from there anyways.That was the reason for its removal